app/models/article.rb

```diff
@@ -100,6 +100,9 @@ class Article < ApplicationRecord
   after_update_commit :update_notifications, if: proc { |article|
                                                    article.notifications.any? && !article.saved_changes.empty?
                                                  }
+  after_update_commit :reindex_comments_to_elasticsearch, if: proc { |article|
```

Comment:
We actually have a [whole syncing pattern setup for models and their related Elasticsearch docs](https://github.com/forem/forem/pull/7049/files). I would lean towards using that same pattern here so that we stick with convention. This will also allow us to update comments for other possible reasons in the future if we need to.

Reply:
Thank you! TIL! That's such a nice pattern. I'm also adding 'title' to the sync reasons too.